### PR TITLE
Update the experimental emojis and some basic readme content

### PR DESCRIPTION
**Pull request by andrewleader**
In Win11, the emojis we were using to indicate experimental became "End" emojis, updating that to be a refresh symbol that's hopefully refresh on both Win10 and Win11 (I don't have a Win10 device to test with anymore though).

Also made a couple minor updates to the readme to ensure we weren't promising 100% incremental usage for 1.0 since XAML Islands is no longer part of 1.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,31 +9,33 @@
...
+* Additive platform APIs *(only add what you need, leave the rest of your app as-is)*
 * Works with Win32, WPF, WinForms, and more apps
 
 Plus, **create beautiful, modern apps for Windows 11 that are backwards compatible** to Windows 10 October 2018 Update (build 17763 / version 1809)!
 
 
 ### Use it now!
-Desktop apps can [use the 0.8 release](https://docs.microsoft.com/windows/apps/windows-app-sdk/stable-channel#version-08) now!
+
+1.0 Preview 3 is [available to try today](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app) and includes unpackaged support including WinUI 3, single-project templates, and more!
```
Comment (riverar) [inline README.md:20]: Bit of a tongue (or eye?) twister here.
```suggestion
1.0 Preview 3 is [available to try today](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app) and includes unpackaged support, single-project templates, and more!
```

File: README.md [hunk 2]
```diff
@@ -9,31 +9,33 @@
...
+
+In production apps, you can [use the 0.8 release](https://docs.microsoft.com/windows/apps/windows-app-sdk/stable-channel#version-08) now!
 
 Some notes about 0.8...
 
-* 0.8 Stable requires using [MSIX](https://docs.microsoft.com/windows/msix/overview), but unpackaged apps can [try the 0.8 Experimental release](https://docs.microsoft.com/windows/apps/windows-app-sdk/deploy-unpackaged-apps)!
+* 0.8 Stable requires using [MSIX](https://docs.microsoft.com/windows/msix/overview), but unpackaged apps can [1.0 Preview 3](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app)!
```
Comment (riverar) [inline README.md:26]: Missing word(s) here.

```suggestion
* 0.8 Stable requires using [MSIX](https://docs.microsoft.com/windows/msix/overview), but unpackaged apps must use [1.0 Preview 3](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app)!
```

**Comment by riverar** [comment]
Heads up: 🔄 is supposed to be `U+1F501 | CLOCKWISE RIGHTWARDS AND LEFTWARDS OPEN CIRCLE ARROWS`. You might want to file a bug internally on Edge, font team, or all of the above.

**Comment by andrewleader** [comment]
Ah thanks @riverar, yeah it seemed odd and incorrect that our emojis ended up like that, I'll try to find the right contacts about that!

**Comment by riverar** [comment]
@andrewleader If you need a bug vehicle, https://aka.ms/AAeki3c

**Review by btueffers — APPROVED** [review]
Looking good aside from @riverar's feedback!

**Comment by andrewleader** [comment]
Thanks! I'll merge now, I'm also working with the emoji team, I heard that they have some emoji changes they're aware of and am confirming if this previous icon was one of them, but for now switching to this other icon.

**Comment by riverar** [comment]
@andrewleader You just merged in typos that I pointed out in review.

**Comment by andrewleader** [comment]
Agh idk how I missed those comments, sorry and thank you! #1701

**Pull request merged**